src/ert/storage/local_ensemble.py

```diff
@@ -208,7 +208,9 @@ def load_parameters(
         return self._load_dataset(group, realizations)[var]
 
     @lru_cache  # noqa: B019
-    def load_response(self, key: str, realizations: Tuple[int, ...]) -> xr.Dataset:
+    def load_responses(
+        self, key: str, realizations: npt.NDArray[np.int_]
+    ) -> xr.Dataset:
```

Comment:
There was also a point in the issue about unifying the behaviour  of the two functions. Øyvind said that `load_parameters` will return an empty dataset while `load_response` will throw a `KeyError`. We can unify it in the same PR or make a new one I guess

Reply:
They both do return KeyError just have a look at the added tests

Reply:
Sorry, my bad